sdk/src/main/java/io/opentelemetry/sdk/trace/RandomIdsGenerator.java

```diff
@@ -22,8 +22,6 @@
 
 final class RandomIdsGenerator implements IdsGenerator {
   private static final long INVALID_ID = 0;
-  // TODO change with ThreadLocal version
```

Comment:
We still don't have this one solved, no?

Reply:
Nope, because Android.